protected/humhub/modules/space/widgets/HeaderControlsMenu.php

```diff
@@ -91,7 +91,7 @@ public function init()
         if ($this->space->isMember()) {
             $membership = $this->space->getMembership();
 
-            if (!$membership->send_notifications) {
+            if (!$membership->send_notifications && !Yii::$app->notification->hasSpace($this->space)) {
```

Comment:
Do we still need the `!$membership->send_notifications` check here?

Reply:
Yes, it may be deleted, If we will remove the `!$membership->send_notifications` from here the result is not changed because inside the method `hasSpace()` this checking is done too but there is an additional sql query to the table `space_memberhip` for all spaces.
I have decided to keep it here only to avoid the sql query when this setting is already enabled per current User and Space.
I.e. the removing will make the code cleaner but a bit slower.